tests/command_submit.py

```diff
@@ -250,3 +250,38 @@ def test_call_submit_worldcodesprint_mars_exploration(self):
         ojtools = os.path.abspath('oj')
         with tests.utils.sandbox(files):
             subprocess.check_call([ojtools, 's', '-y', '--no-open', url, 'a.py'], stdout=sys.stdout, stderr=sys.stderr)
+
+class SubmitTophTest(unittest.TestCase):
+    @unittest.skipIf('CI' in os.environ, 'login is required')
+    def test_call_submit_copycat(self):
+        url = 'https://toph.co/p/copycat'
+        code = '''#!/usr/bin/env python3
+s = input()
+print(s)
+'''
+        files = [
+            {
+                'path': 'a.py',
+                'data': code
+            },
+        ]
+        ojtools = os.path.abspath('oj')
+        with tests.utils.sandbox(files):
+            subprocess.check_call([ojtools, 's', '-l', '58482c1804469e2585024324', '-y', '--no-open', url, 'a.py'], stdout=sys.stdout, stderr=sys.stderr)
+
+    @unittest.skipIf('CI' in os.environ, 'login is required')
+    def test_call_submit_add_them_up(self):
```

Comment:
(not so important) It becomes better that this test case should submit C++ code, instead of Python code.

This tool guesses language ids (e.g. `584859c204469e2585024499` `55b4af33421aa95e80000001` ...) from language names (e.g. `C++14` `C++11` `C++` `Python 3` `PyPy 3` ...). The language names are different for each services and the guessing sometimes fails, so this case should also test this for C++.